larray/viewer.py

```diff
@@ -1985,8 +1990,13 @@ def setup_and_check(self, data, title='', readonly=False,
             title += ' (' + _('read only') + ')'
         self.setWindowTitle(title)
 
+        self.statusBar().showMessage("Welcome to the LArray Viewer", 4000)
+
+        widget = QWidget()
+        self.setCentralWidget(widget)
```

Comment:
http://doc.qt.io/qt-4.8/qmainwindow.html

Below image in section 'Qt Main Window Framework'

Note: Creating a main window without a central widget is not supported. You must have a central widget even if it is just a placeholder.